sw/device/silicon_creator/rom/data/rom_testplan.hjson

```diff
@@ -474,10 +474,13 @@
 
             `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
 
-            - For `command` in {`SECTOR_ERASE` (`0xc7`) and `PAGE_PROGRAM (`0x02`)}
+            - For `command` in {`SECTOR_ERASE` (`0xc7`) and `PAGE_PROGRAM` (`0x02`)}
               - Apply bootstrap pin strapping and reset the chip.
-              - Send `command` with invalid 3-byte address, e.g. `0xffffff`.
+              - Send `CHIP_ERASE` command to transition to bootstrap phase 2.
+              - Send `command` with an invalid 3-byte address, e.g. `0xffffff`.
               - Verify that the chip outputs the expected `BFV` over UART.
+                - For `SECTOR_ERASE`: `BFV:01425303` (`kErrorBootstrapEraseAddress`) followed by `BFV:0142500d` (`kErrorBootPolicyBadIdentifier`)
+                - For `PAGE_PROGRAM`: `BFV:02425303` (`kErrorBootstrapProgramAddress`) followed by `BFV:0142500d` (`kErrorBootPolicyBadIdentifier`)
```

Comment:
note: astute reviewers may have noticed that these values don't match the ones used in the test. I'm working on a PR for cleaning up error values (#14774). I kept the new values here since I already updated the issue description  in #14467.

Reply:
Done